.github/workflows/ci.yaml

```diff
@@ -0,0 +1,105 @@
+name: Helm Chart Package CI
+on:
+  push:
+    branches:
+      - 'master'
+      - 'cert-manager-feature-branch'
+    tags:
+      - 'v*.*.*'
+  pull_request:
+    branches:
+      - 'master'
+      - 'cert-manager-feature-branch'
+
+jobs:
+  # pre job run golangci-lint
+  go-lint:
+    name: 'Golint'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.30
+          working-directory: ./cmd/
+
+  # pre job to run helm lint
+  helm:
+    name: HelmLint
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Get repositories
+        uses: hiberbee/github-action-helm@latest
+
+      - name: Lint chart
+        run: helm lint cockroachdb
+        working-directory: .
+
+  # pre job to run the unit tests
+  unitTest:
+    name: UnitTest
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+
+      - name: Unit
+        run: go test -v ./cmd/...
+
+      - name: HelmTemplate
+        run: go test -v ./tests/template/...
+
+  # pre job to build and push the self signer utility
+  pre-build:
+    name: build-self-signer-cert-utility
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GCR
+        uses: docker/login-action@v1
+        with:
+          registry: gcr.io
+          username: _json_key
+          password: ${{ secrets.GCR_HELM_CHART_SA_JSON` }}
```

Comment:
Need to get rid of the ` character.

Are you sure you can use the docker login flow and just pass in the JSON cert file? I don't think this will work, have you tested it?

Reply:
I have fixed this typo and verified that it works on GCR repo. Link for my fork where it is working: https://github.com/prafull01/helm-charts/pull/4